docker/1.4.0/final/Dockerfile.cpu

```diff
@@ -22,13 +22,13 @@ RUN cd /tmp && \
 
 WORKDIR /
 
-COPY dist/sagemaker_mxnet_container-2.0.0.tar.gz /sagemaker_mxnet_container-2.0.0.tar.gz
+COPY dist/sagemaker_mxnet_container-3.0.0.tar.gz /sagemaker_mxnet_container-3.0.0.tar.gz
 
 RUN pip install --no-cache mxnet-mkl==1.4.0.post0 \
```

Comment:
apologies, this isn't in your PR scope, but I think this should be pinned at 1.4.0 and not post as stated here: https://github.com/aws/sagemaker-mxnet-container/pull/79#discussion_r281397147

Reply:
should I run a git blame... 😂 

(fixed)